server.js

```diff
@@ -287,7 +287,7 @@ function getMockConfigData() {
         },
         {
           spec: {
-            clusterServiceClassExternalName: 'amq-broker-72-persistence'
+            clusterServiceClassExternalName: 'amq-online-standard'
```

Comment:
Looks like there's two `amq-online-standard`s listed here now?

Reply:
right, i'll remove that. It's the mock config, so can be verified with `yarn start:dev`